accounts/templates/partials/_reading_goal.html

```diff
@@ -0,0 +1,142 @@
+                "Update" button. Otherwise, press ESC key or click on ✕ button to close</p>
+            <form id="update-form" class="flex flex-col gap-4" hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'>
+                <input type="text" placeholder="Current Goal: {{ goal.goal }}" name="goal"
+                    class="input input-bordered w-full" required />
+                <button class="btn btn-primary" type="submit" hx-post="{% url 'update_goal' %}"
+                    hx-target="#reading_goal_section" hx-include="#update-form" hx-swap="outerHTML">Update
+                    Resolution</button>
+            </form>
+        </div>
+    </dialog>
+    <!-- #endregion Edit Modals -->
+    <!-- #region Delete Modal -->
+    <dialog id="delete_modal" class="modal">
+        <div class="flex flex-col modal-box">
+            <form method="dialog">
+                <button class="btn btn-sm btn-circle btn-ghost absolute right-2 top-2">✕</button>
+            </form>
+            <h3 class="text-lg font-bold">Are you sure?</h3>
+            <p class="py-4">Pressing the confirm button will delete your Reading Resolution and ALL
+                associated books you
+                have read. Press CONFIRM to DELETE, otherwise press ESC key or click on ✕ button to close
+                and keep
+                going.</p>
+            <button class="btn btn-error" hx-get="{% url 'delete_goal' %}" hx-target="#reading_goal_section"
+                hx-swap="outerHTML">DELETE RESOLUTION</button>
+        </div>
+    </dialog>
+    <!-- #endregion Delete Modals -->
+    <!-- #endregion Modals -->
+    <script src="{% static '/js/goal_search.js' %}"></script>
+</section>
+{% else %}
+<!-- No Reading Resolution -->
+<section id="reading_goal_section" class="flex flex-col max-md:w-10/12 mx-auto my-10 space-y-12">
+    <div class="flex flex-col bg-[#6f42c166] rounded-3xl">
+        <div class="flex flex-col m-8">
+            <p class="text-5xl font-bold my-2 mb-5">Book Club Reading Resolution!</p>
+            <p>Get Smarter Everyday by joining the the Book Club Reading Resolution! The Book Club Reading Resolution is
+                a self-paced reading program designed to make reading fun, interactive, supportive. Set a goal for the
+                year and track each book you read, whether its with the book club or on your own! Each book you read
+                will earn yourself a 'Little A' sticker to go on a printable PDF Certificate and track your progress in
+                your profile in the Big A Book Club website. Reach your reading goal by the end of the year and earn a
+                special prize!</p>
+            <button hx-get="{% url 'create_goal' %}" hx-target="#reading_goal_section" hx-swap="outerHTML"
+                class="btn btn-wide btn-large btn-outline btn-neutral my-6 mx-auto text-lg">Start Reading
+                Resolution</button>
```

Comment:
+1 if you could [incorporate localization](https://docs.djangoproject.com/en/5.1/topics/i18n/) for best practices

Reply:
Will address in new PR, please create an issue to track for future development